src/plugins/mwdb_uploads.py

```diff
@@ -5,24 +5,24 @@
 
 from mwdblib import Malwarecage
 
-from metadata import Metadata, MetadataPlugin
+from db import Database
+from metadata import Metadata, MetadataPlugin, MetadataPluginConfig
 
 
 class MalwarecageUploadsMetadata(MetadataPlugin):
-    """
-    :param mwdb_api_token: API key for 'mquery' user in Malwarecage
-    :param mwdb_api_url: API URL accessible from mquery daemon
-    :param mwdb_url: Malwarecage URL accessible for mquery users
-    """
-
     __cacheable__ = True
-
-    def __init__(
-        self, mwdb_api_token: str, mwdb_api_url: str, mwdb_url: str
-    ) -> None:
-        super().__init__()
-        self.mwdb = Malwarecage(api_url=mwdb_api_url, api_key=mwdb_api_token)
-        self.mwdb_url = mwdb_url
+    __config_fields__ = {
+        "mwdb_url": "Malwarecage URL accessible for mquery users",
+        "mwdb_api_url": "API URL accessible from mquery daemon",
```

Comment:
```suggestion
        "mwdb_api_url": "API URL to malwarecage instance (usually mwdb_url + '/api/')",
```

Reply:
Also *technically* we could use a reasonable default here, but maybe it's not worth the effort.